integration-test/spark-common/src/test/java/com/tencent/rss/test/SparkClientWithLocalTest.java

```diff
@@ -50,6 +43,14 @@
 import org.roaringbitmap.longlong.LongIterator;
 import org.roaringbitmap.longlong.Roaring64NavigableMap;
 
+import java.io.File;
```

Comment:
Need we a mores strict `import  order` in code style checker?

Reply:
The rule of import order should be added to project configuration.

Reply:
@frankliee Can you add these rules?

Reply:
I will do it.

Reply:
Use iceberg rule https://github.com/apache/iceberg/blob/master/.baseline/checkstyle/checkstyle.xml#L247 @frankliee

Reply:
Iceberg's style is wired.

import com.tencent.rss.common.Arguments;
import com.tencent.rss.common.metrics.GRPCMetrics;
import com.tencent.rss.common.metrics.JvmMetrics;
import com.tencent.rss.common.rpc.ServerInterface;
import com.tencent.rss.common.web.CommonMetricsServlet;
import com.tencent.rss.common.web.JettyServer;
import io.prometheus.client.CollectorRegistry;
import java.io.FileNotFoundException;
import org.slf4j.Logger;
import org.slf4j.LoggerFactory;
import picocli.CommandLine;

Reply:
In fact, spark disables its java import checker....